src/app/clusters/ota-requestor/OTARequestor.cpp

```diff
@@ -149,20 +150,23 @@ void OTARequestor::OnQueryImageResponse(void * context, const QueryImageResponse
             ChipLogDetail(SoftwareUpdate, "Version %" PRIu32 " is older or same than current version %" PRIu32 ", not updating",
                           update.softwareVersion, requestorCore->mCurrentVersion);
 
-            requestorCore->mOtaRequestorDriver->UpdateNotFound(UpdateNotFoundReason::UpToDate,
-                                                               System::Clock::Seconds32(response.delayedActionTime.ValueOr(0)));
+            requestorCore->mOtaRequestorDriver->UpdateNotFound(
```

Comment:
[GenericOTARequestorDriver.cpp](https://github.com/project-chip/connectedhomeip/blob/master/src/platform/GenericOTARequestorDriver.cpp#L61-L68) does enforce the minimum delay of 120 in `UpdateNotFound` and `UpdateSuspended` APIs.